build.gradle

```diff
@@ -96,7 +96,7 @@ subprojects {
   project(':Apromore-Boot') {
 
     bootJar.baseName = "Apromore-Core"
-    ext['user_home'] = System.properties['user.home']
+    ext['user_home'] = System.properties['user.home'].replaceAll("\\\\", "/")
```

Comment:
Compare to the ApromoreEE equivalent fix on line 166 of https://github.com/apromore/ApromoreEE/blob/development/build.gradle